From ae3464a82e1395fc577cc94999bdc7c4a7bb35f1 Mon Sep 17 00:00:00 2001
From: Dmytro Kostiuchenko <edio@archlinux.us>
Date: Thu, 25 Jun 2020 11:08:26 -0400
Subject: [PATCH] Fix compilation errors

This commit applies a patch submitted in #5 and initially offered at
https://mail.gnome.org/archives/gimp-developer-list/2020-June/msg00004.html
---
 src/interface_I.c   | 10 +++++-----
 src/interface_aux.c |  8 ++++----
 2 files changed, 9 insertions(+), 9 deletions(-)

diff --git a/src/interface_I.c b/src/interface_I.c
index b680a2f..5e04754 100644
--- a/src/interface_I.c
+++ b/src/interface_I.c
@@ -65,13 +65,13 @@ static void callback_alarm_triggered (GtkWidget * size_entry, gpointer data);
 
 gint dialog_I_response = GTK_RESPONSE_OK;
 
-PlugInUIVals *ui_state;
-PlugInVals *state;
-PlugInDialogVals *dialog_state;
-gboolean features_are_sensitive;
+extern PlugInUIVals *ui_state;
+extern PlugInVals *state;
+extern PlugInDialogVals *dialog_state;
+extern gboolean features_are_sensitive;
 InterfaceIData interface_I_data;
 
-GtkWidget *dlg;
+extern GtkWidget *dlg;
 GtkWidget *coordinates;
 
 gulong size_changed = 0;
diff --git a/src/interface_aux.c b/src/interface_aux.c
index a6bcc8d..58d0af4 100644
--- a/src/interface_aux.c
+++ b/src/interface_aux.c
@@ -48,11 +48,11 @@ static void callback_dialog_aux_response (GtkWidget * dialog, gint response_id,
 
 gint dialog_aux_response = GTK_RESPONSE_OK;
 
-PlugInUIVals *ui_state;
-PlugInVals *state;
-PlugInDialogVals *dialog_state;
+extern PlugInUIVals *ui_state;
+extern PlugInVals *state;
+extern PlugInDialogVals *dialog_state;
 
-GtkWidget *dlg;
+extern GtkWidget *dlg;
 
 /***  Public functions  ***/
 
